client/styles/pages/_tests-runs.scss

```diff
@@ -8,6 +8,7 @@
         #noData {
```

Comment:
Do not use `#` selector without an urgent need for it. Please refactor this selector.